File: LeetCode/70.Climbing_Stairs/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    int t1 = 2;
    int t2 = 3;
    if (s.climbStairs(t1) == 2) { cout << "Pass" << endl;}
    else { cout << "Failed 1 - t1" << endl;}
    if (s.climbStairs(t2) == 3) { cout << "Pass" << endl;}
    else { cout << "Failed 2 - t2" << endl;}
    return 0;
}

File: LeetCode/70.Climbing_Stairs/main.py
#!/usr/bin/python

from solution import Solution

# Unit Test
t1 = 2
t2 = 3
s = Solution()
if (s.climbStairs(t1)==2):
    print("Pass")
else:
    print("Test 1 Failed")
if (s.climbStairs(t2) == 3):
    print("Pass")
else:
    print("Test 2 Failed")